src/numbuf/python/src/pynumbuf/adapters/python.cc

```diff
@@ -80,6 +80,25 @@ Status get_value(std::shared_ptr<Array> arr, int32_t index, int32_t type, PyObje
   return Status::OK();
 }
 
+Status call_custom_serialization_callback(PyObject* elem, PyObject** serialized_object) {
```

Comment:
The naming convention for functions returning Statuses is CallCustomSerializationCallback